kytos/utils/decorators.py

```diff
@@ -65,6 +65,21 @@ def authenticate(self):
         username = self.config.get('auth', 'user')
         password = getpass("Enter the password for {}: ".format(username))
         response = requests.get(endpoint, auth=(username, password))
+        if response.status_code == 401:
+            invTokenStr = "{\"error\":\"Token not sent or expired: " \
+                          "Signature has expired\"}\n"
+            # pylint: disable=superfluous-parens
+            if (invTokenStr == response.content.decode()):
```

Comment:
I think these lines can be safely removed. Line 68 (http status code 401) alone needs user re-authentication.